resources/install/example.sql

```diff
@@ -19,3 +19,14 @@ CREATE TABLE IF NOT EXISTS `activities_activities` (
     KEY `row` (`row`),
     KEY `ip` (`ip`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE IF NOT EXISTS `activities_resources` (
```

Comment:
@amazeika For the textman revisions table we are using the following db schema

https://github.com/joomlatools/textman/blob/master/code/administrator/components/com_textman/resources/install/install.sql

Note the addition of a uuid column and the use of article_id vs row, here we could use resource_id instead of row, i'm trying to get rid of the row column as this is too generic. 

For the metadata see my previous comment.